packages/ndla-error-reporter/src/__tests__/ErrorReporter-test.js

```diff
@@ -38,12 +38,12 @@ test('ndla-error-reporter/ErrorReporter can capture message', () => {
     })
     .reply(200);
 
-  errorReporter.captureMessage('Log message');
+  await errorReporter.captureMessage('Log message');
```

Comment:
`captureMessage` må faktisk returnere noe for at `nock.done` skal vente på at alle promises er oppfylt.